Sources/Encoder.swift

```diff
@@ -465,4 +465,37 @@ public struct Encoder {
         }
     }
     
+    /**
+     Sets value for provided key path delimited by provided delimiter.
+     
+     - parameter valueToSet:    Value to set
+     - parameter keyPath:       Key path.
+     - parameter withDelimiter: Delimiter for key path.
+     */
+    public static func setValue(dict: inout JSON, value: AnyObject, forKeyPath: String, withDelimiter: String = GlossKeyPathDelimiter) {
```

Comment:
i think that Encoder already has its clear responsibility - it transforms input values into JSON. this `setValue` function doesn't inherently have anything to do with that - we just happen to know its used for that purpose. it's more like a JSON utility function that we use for encoding.

this functionality was before in the `Dictionary` extension and it made more sense to me there